packages/wholly-sheet/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+### [0.5.1](https://www.github.com/looker-open-source/sdk-codegen/compare/wholly-sheet-v0.5.0...wholly-sheet-v0.5.1) (2021-03-30)
+
+
+### Features
+
+* Added --versions option to the code generator ([#514](https://www.github.com/looker-open-source/sdk-codegen/issues/514)) ([ee6f3e8](https://www.github.com/looker-open-source/sdk-codegen/commit/ee6f3e8f55e300df1a75c9be89b47f067bc08dee))
```

Comment:
Weird this seems to be mostly in related packages rather than the codegen packages. I probably did something wrong with my commits?